src/pyfvtool/cell.py

```diff
@@ -285,7 +435,8 @@ def cellLocations(m: MeshStructure):
     
     It can later be used in defining properties that are variable in space.
     
-    Incompletely tested
+    Incompletely tested, and there may be other, more direct, ways to 
+    calculate properties that vary in space, e.g. using cellcenters directly?
```

Comment:
What I meant with the 'more direct ways', is that we may simply rely on NumPy arithmetic to do this (and then fill the CellVariable with the required values), without adding extra infrastructure to PyFVTool, keeping it lean and simple. 
But there may indeed be efficient functions/methods to be developed that make life easier, and the user code clearer.

Reply:
I thought about making cells a subclass of `numpy` arrays but then decided against it. The chemical engineer inside me told me that it can become too complicated.